src/test/scala/TableTest.scala

```diff
@@ -300,4 +301,51 @@ class TableTest extends FunSuite {
       thrown.getMessage === "@type of table is not 'Table' - country_slice.csv.@type"
     )
   }
+
+  test(
+    "sets scala test for Array[Any] and List[Any]"
```

Comment:
Would be good to explain why this test is here and what it demonstrates.

Reply:
Done